src/components/NationalText/utils.ts

```diff
@@ -0,0 +1,62 @@
+import AggregationsJSON from 'assets/data/aggregations.json';
+import { last } from 'lodash';
+import moment from 'moment';
+import { formatPercent, formatDecimal } from 'common/utils';
+
+// Fix all uses of !
+
+const getPercentChange = (series: any[]): number => {
+  const lastTwoWeeks = series.slice(-14);
+  const difference = last(lastTwoWeeks)! - lastTwoWeeks[0]!;
+  const percentChange = difference / lastTwoWeeks[0]!;
+  return percentChange;
```

Comment:
Consider finding the point that matches the date instead of relying on the series having all the points and being sorted by date, this will help us in case there are holes in the data

```ts
const currPoint = last(sortedSeries);
const prevDate = moment(currPoint.x).subtract(14, 'days').toDate();
const prevPoint = sortedSeries.find(p => isSameDay(p.x, prevDate));

return prevPoint && currPoint ? precentChange(prevPoint.y, currPoint.y) : null;

// ...

// maybe it would be better to compare `moment(date).format('YYYY-MM-DD')`
function isSameDay(dateA: Date, dateB: Date) {
  return moment(dateA).startOf('day') === moment(dateB).startOf('day');
}

```